templates/layout.html

```diff
@@ -27,33 +27,37 @@
         <ul class="nav navbar-nav">
           {% for href, title in [
             ['/about', 'About'],
+            ['/blog', 'Blog'],
             ['/media', 'Media'],
             ['/downloads', 'Downloads'],
-            ['/blog', 'Blog']
+            ['/get-involved', 'Get involved'],
+            ['/support/', 'Support']
           ] %}
             <li{% if this.is_child_of(href) %} class="active"{% endif
               %}><a href="{{ href|url }}">{{ title }}</a></li>
           {% endfor %}
+          <li><a href="https://github.com/unknown-horizons/unknown-horizons/wiki/Donation-History" class="donate">Donate</a></li>
```

Comment:
Do we really need a link to the donation history here? It says

> Thanks for your helpfulness, we don't need money right now to improve the game.

right at the top, so I don't think it needs such a prominent placing (if any at all). Also, the page it links to appears to be very broken.